src/tmducken/duckdb.clj

```diff
@@ -105,14 +105,14 @@ _unnamed [5 3]:
             (when-not is-init?
               (let [duckdb-home (or duckdb-home
                                     (System/getenv "DUCKDB_HOME")
-                                    "./binaries")
+                                    (str "/" com.sun.jna.Platform/RESOURCE_PREFIX))
```

Comment:
This is a breaking change, but probably only for local environments.
With the other changes in this commit, if you use `nix shell` or `direnv` the duckdb library gets put in the correct place automatically, so it doesn't break any existing flows.

Reply:
It loads like this now:
```
INFO: Attempting to load duckdb from "/darwin-aarch64/libduckdb.dylib"
```

Reply:
I love it - we will want to work this in for sure.  I think it is fine actually as local defs can be sure to set DUCKDB_HOME - we always are.